src/metpy/calc/kinematics.py

```diff
@@ -128,7 +128,7 @@ def vorticity(u, v, dx, dy):
 @exporter.export
 @preprocess_xarray
 @ensure_yx_order
-@check_units('[speed]', '[speed]', '[length]', '[length]')
+@check_units(dx='[length]', dy='[length]')
 def divergence(u, v, dx, dy):
     r"""Calculate the horizontal divergence of the horizontal wind.
```

Comment:
```suggestion
     r"""Calculate the horizontal divergence of two vectors.
```

Reply:
Done and done. I saw that late Saturday too after I pushed the original commit...just didn't have the energy to do that then, then I forgot yesterday. Thanks for pointing it out!

Reply:
Thanks!